fe/fe-core/src/main/java/org/apache/doris/analysis/SlotRef.java

```diff
@@ -74,9 +74,6 @@ public SlotRef(SlotDescriptor desc) {
         this.type = desc.getType();
         // TODO(zc): label is meaningful
         this.label = null;
-        if (this.type.equals(Type.CHAR)) {
```

Comment:
Are you sure this is OK to remove this?

Reply:
already do some test, but I am not sure for all case, because the motivation to transfer char to varchar in unknown for me

Reply:
If it is unknown, better not modify it. Or it is very hard to track.
And please do not do 'bug fix' in this PR, because the title of this PR is `[fix](ut)`

Reply:
ok